src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32.cs

```diff
@@ -32,10 +32,16 @@ public class Win32 : IProgram
         public bool Enabled { get; set; }
         public bool hasArguments { get; set; } = false;
         public string Location => ParentDirectory;
+        public string AppType { get; set; }
 
         private const string ShortcutExtension = "lnk";
         private const string ApplicationReferenceExtension = "appref-ms";
         private const string ExeExtension = "exe";
+        private const string InternetShortcutExtension = "url";
+
+        private const string InternetShortcutApplication = "Internet Shortcut application";
+        private const string Win32Application = "Win32 application";
+        private const string ProgressiveWebApplication = "Progressive Web application";
```

Comment:
Should we add "/Pinned Web site" because not every pinned web site is a PWA? Not shure how MS Edge and Chrome call it.

Is there a way to show the browser like "Installed web site/PWA running via: msedge.exe"?

Reply:
@crutkas, @jyuwono, any thoughts on what it should be named? Right now I went with `Progressive Web application`. 
![image](https://user-images.githubusercontent.com/28739210/84190592-0963e700-aa4c-11ea-954d-128e5ab110e4.png)